src/api/spec/controllers/attribute_controller_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe AttributeController, type: :controller do
+  render_views
+
+  describe '#update' do
+    let!(:admin) { create(:admin_user) }
+    let(:user1) { create(:confirmed_user) }
+    let!(:namespace) { create(:attrib_namespace, name: 'TEST') }
+    let(:attribute) { create(:attrib_type, attrib_namespace: namespace) }
+    let(:dummy) { 'Dummy' }
```

Comment:
Huh?

Reply:
this is merely taken from the minitest

Reply:
Ah okay, can you remove it anyway? :-)

Reply:
no, as the variable (or whatever you call these lets) is used all over in the test :)

Reply:
I thought you know `%:s`...